CustomApps/new-releases/index.js

```diff
@@ -100,7 +102,13 @@ class Grid extends react.Component {
 		this.setState({ cards: [...gridList] });
 	}
 
-	async reload() {
+	removeCards(id) {
+		dismissed = dismissed[0] === null ? [id] : dismissed.concat(id);
```

Comment:
Array variables are mutable.
```suggestion
		dismissed.push(id);
```

Reply:
this had a reason since it would append after a null value if the list was empty, could be an oversight in how i was fetching localstorage however and your suggestions above could fix it.